cmd/template/cluster/flag.go

```diff
@@ -25,10 +26,11 @@ const (
 	flagDescription         = "description"
 	flagMasterAZ            = "master-az" // TODO: Remove some time after August 2021
 	flagName                = "name"
-	flagOutput              = "output"
-	flagOwner               = "owner"
-	flagRelease             = "release"
-	flagLabel               = "label"
+	// TODO: Fix this - the genericclioptions already have an output field. We can't redefine flags.
+	flagOutput  = "output-gs"
```

Comment:
so what would the `output` flag do now?

Reply:
The overwrite magic thingy which we used shared in kubectl-gs has already an output flag set. I still need to clean this up somehow but will talk to @axbarsan to make it nice.

Reply:
This should be all fixed now.